python/xorbits/_mars/dataframe/reduction/aggregation.py

```diff
@@ -861,6 +870,32 @@ def _execute_agg(cls, ctx, op: "DataFrameAggregate"):
             concat_df = concat_df.astype(op.outputs[0].dtypes, copy=False)
         ctx[op.outputs[0].key] = concat_df
 
+    @classmethod
+    def _cudf_agg(cls, op: "DataFrameAggregate", in_data):
+        data = in_data
+        if is_series(in_data):
+            data = in_data.to_frame()
+            # cudf to_frame doc: https://docs.rapids.ai/api/cudf/stable/api_docs/api/cudf.series.to_frame
+            # The `name` option is None by default, but the actual behaviour sets the name to `0`
+            # So define the columns manually
+            data.columns = [in_data.name]
+
+        func_name = (
+            op.func_rename[0] or op.func[0] if len(op.func) == 1 else op.func_rename
+        )
+        # cudf do not support that axis is not None
+        # when axis is 0, make it None
+        # when axis is not None, just raise Error
+        # ref: https://docs.rapids.ai/api/cudf/stable/api_docs/api/cudf.dataframe.agg
+        axis = op.axis if op.axis else None
```

Comment:
For better readability, how about
```suggestion
        axis = op.axis if op.axis == 1 else None
```